cli/lib/utils.ts

```diff
@@ -50,15 +52,18 @@ const RULE_AUTOGENERATED_COMMENT = `
   ${padEnd('//', 80, '/')}
 `
 
-/** Generate the role script text */
-export async function generateRuleScript(ruleDef: RuleDefinition) {
+/** Generate the code of the Auth0 script as a string */
+export async function generateScript(
+  ruleDef: RuleDefinition | DBActionScriptDefinition,
+  scriptType: 'rules' | 'db' = 'rules'
```

Comment:
This might be the only place you use it, but I feel like the union type should be defined somewhere reusable (`export type ScriptType = 'rules' | 'db'`)

Reply:
Also not sure I love the default parameter, feels like you should specify it explicitly.

Reply:
I was kinda thinking that Rules would be the first class citizen of this repo, and DB Action Scripts would be an addition. However, it's not like someone likely to be writing a new call of generateScript in diff or deploy very frequently, and it might be nice to see it explicitly.